pywbem_mock/_wbemconnection_mock.py

```diff
@@ -0,0 +1,1571 @@
+#!/usr/bin/env python
```

Comment:
Modules should not have a hashbang. (only scripts should)

Reply:
Done.

Reply:
DONE